helpers/database-import-scripts/rnacentral/generate_rnacentral_json.py

```diff
@@ -428,16 +507,30 @@ def get_project_accession(biosample):
 
 
 def load_xml(sample_id, insdc_accession=None):
+    if sample_id in ERROR_404:
+        logging.warning("Not attempting to retrieve xml for sample {}. Reason: sample is not in ENA".format(
+            sample_id))
+        return None
     xml_url = 'https://www.ebi.ac.uk/ena/browser/api/xml/{}'.format(sample_id)
     try:
         r = run_xml_request(xml_url)
     except:
-        if insdc_accession.startswith("C"):
-            logging.error("Unable to get XML for sample {}, ENA genome {}".format(sample_id, insdc_accession))
-            sys.exit()
+        if r is None:
+            if insdc_accession.startswith("C"):
+                logging.error("Unable to get XML for sample {}, ENA genome {}".format(sample_id, insdc_accession))
+                sys.exit()
```

Comment:
Should the script exit here? Should just print the error and skip the genome?

Reply:
It should exit because this means something is wrong with the ENA request. There might be some edge case where it's not an ENA issue but I haven't encountered that yet and it's better to deal with it properly if it comes up.